objc/A0LockReactModule.m

```diff
@@ -97,10 +97,16 @@ @implementation A0LockReactModule
     });
 }
 
+RCT_EXPORT_METHOD(getDelegationToken:(NSDictionary *)options callback:(RCTResponseSenderBlock)callback) {
```

Comment:
Let's call it just `delegation:callback` so in JS is just `delegation(options, callback)`